src/components/diff/NotebookDiff.ts

```diff
@@ -360,12 +373,20 @@ namespace Private {
   /**
    * Marks certain cells with
    */
-  export function markUnchangedRanges(root: HTMLElement): void {
-    const children = root.querySelectorAll(`.${CELLDIFF_CLASS}`);
+  export function markUnchangedRanges(
+    root: HTMLElement,
+    isConflict: boolean
+  ): void {
+    const CELL_CLASS = isConflict ? CELLMERGE_CLASS : CELLDIFF_CLASS;
+    const UNCHANGED_CLASS = isConflict
+      ? UNCHANGED_MERGE_CLASS
+      : UNCHANGED_DIFF_CLASS;
+
```

Comment:
This should be the fix for merge notebooks and the `hide unchanged cells` option, but the button does nothing for me, even on `master`. 

This could be my notebook though, and I'm probably mistaking an unchanged cell, with a cell that has *unchanged output* as the same thing.

Reply:
For the diff view, unchanged cells are cells identical on all parts (input, output and metadata). On the screenshot above it seems that the third cell should fall into that category.